pkg/server/status.go

```diff
@@ -17,14 +17,22 @@ limitations under the License.
 package server
 
 import (
-	"errors"
-
 	"golang.org/x/net/context"
 
 	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1"
 )
 
 // Status returns the status of the runtime.
 func (c *criContainerdService) Status(ctx context.Context, r *runtime.StatusRequest) (*runtime.StatusResponse, error) {
-	return nil, errors.New("not implemented")
+	runtimeReady := &runtime.RuntimeCondition{
+		Type:   runtime.RuntimeReady,
+		Status: true,
+	}
+	networkReady := &runtime.RuntimeCondition{
+		Type:   runtime.NetworkReady,
+		Status: true,
```

Comment:
cri-containerd doesn't support cider allocation now (we are directly using cri-o cni plugin package now).

The networking team is also planing to deprecate kubenet. In the future, the cider should be handled by a daemonset outside of the container.

/cc @freehan